bitcoin_client/ledger_bitcoin/client_command.py

```diff
@@ -3,6 +3,8 @@
 from collections import deque
 from hashlib import sha256
 
+from ledgercomm.log import LOG
+
 from .common import ByteStreamParser, sha256, write_varint
 from .merkle import MerkleTree, element_hash
 
```

Comment:
Probably better to own the local logger

```suggestion
from .common import ByteStreamParser, sha256, write_varint
from .merkle import MerkleTree, element_hash

import logging

LOG = logging.getLogger("ledger_bitcoin.client_command")

```

Reply:
Taken